oak-auth-external/pom.xml

```diff
@@ -187,7 +197,27 @@
         </dependency>
         <dependency>
             <groupId>org.apache.sling</groupId>
-            <artifactId>org.apache.sling.testing.osgi-mock</artifactId>
+            <artifactId>org.apache.sling.testing.osgi-mock.junit4</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.sling</groupId>
+            <artifactId>org.apache.sling.testing.osgi-mock.core</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
```

Comment:
Why all the additional dependencies? Isn't the transitive dependencies from osgi-mock enough?
It is unusual to reference OSGi Spec chapter dependencies only in test classpath. In tests those are only used usually if the to-be tested code relies on those spec chapters as well.

Reply:
The integration tests will fail without them.

Reply:
but all those are transitive ones of https://github.com/apache/sling-org-apache-sling-testing-osgi-mock/blob/45fb63cb61615663e10ed3c9e612a3a3f6161c3e/core/pom.xml#L42 and the ITs don't leverage those APIs/spec chapters.

Reply:
Yeah. But the tests will fails anyway. Do you have a suggestion?